Tests/Unit/NetworkServer/ConcentratorDeduplicationTest.cs

```diff
@@ -108,8 +108,6 @@ public void CreateKeyMethod_Should_Return_Expected_Keys_For_Different_Payloads(s
             var options = fieldNotUsedInKey ?? string.Empty;
             var payload = new LoRaPayloadDataLns(new DevAddr(devAddr), new MacHeader(MacMessageType.ConfirmedDataUp),
                                                  frameCounter, options, rawPayload, new Mic(mic));
-            _ = Hexadecimal.TryParse(rawPayload, out uint raw);
-            payload.RawMessage = BitConverter.GetBytes(raw);
```

Comment:
I don't understand this change - why do we not need to set the raw message any longer and modify the expected key of only one test?

Reply:
I've reverted this with 2622525262a7d3b993c7b8da8cbe1f884c9736fa as it should be addressed by bug #1152 that was identified as part of this PR.